sdk/src/client/node_api/mqtt/types.rs

```diff
@@ -33,8 +33,10 @@ pub struct TopicEvent {
 pub enum MqttPayload {
     /// In case it contains JSON.
     Json(Value),
-    /// In case it contains a `Block` object.
-    Block(BlockDto),
+    // TODO: should we still call this variant `Block`? It's less exact, but the mqtt topic will still be called
+    // `blocks` instead of `block_bodies`.
```

Comment:
For the reviewers ... here I meant: Should we keep calling this variant `Block` instead of the introduced rename to `BlockBody`

Reply:
But it returns the full block and not just the body?

Reply:
just the body ... at least this is what it refactored to.

Reply:
No I mean the event API returns the full blocks and not just the block bodies? So it needs to be blocks I think

Reply:
https://studio.asyncapi.com/?url=https://raw.githubusercontent.com/iotaledger/tips/tip48/tips/TIP-0048/asyncapi3.yaml only read block there, nothing with body or similar

Reply:
Yes I also think it was wrong before

Reply:
Fixed